.github/workflows/main.yaml

```diff
@@ -1,25 +1,41 @@
-# name: tests
-
-# on:
-#   push:
-#     branches: [ master ]
-#   pull_request:
-#     branches: [ master ]
-
-# jobs:
-#   tests:
-#     runs-on: ubuntu-20.04
-#     steps:
-
-#       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-#       - uses: actions/checkout@v2
-
-#       - name: Install deps
-#         run: $GITHUB_WORKSPACE/install_deps.sh -y
-#         shell: sh
-#         env:
-#           DEBIAN_FRONTEND: noninteractive
-
-#       - name: Run tests
-#         run: $GITHUB_WORKSPACE/runner.sh $GITHUB_WORKSPACE/configs/ci.yaml
-#         shell: sh
+name: illixr-tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  illixr-tests:
+    runs-on: ubuntu-18.04
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Get Branch Name
+        run:  echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+        id: extract_branch
+
+      - name: Login to ghcr.io
+        run: echo ${{ secrets.GHCR_TOKEN }} | docker login ghcr.io -u illixr-ci --password-stdin
+
+      - name: Pull Docker Cache
+        run: docker pull --all-tags ghcr.io/illixr/illixr-tests || true
```

Comment:
What is the `true` for?

Reply:
If pulling fails, we can still build this package (without the benefit of the cache).